index.html

```diff
@@ -64,6 +64,148 @@ <h5> LET's CONNECT</h5>
+<button class="button button2">See Project</button>
+    </div>
+
+
+    
+    <div class="items work3">
+        <div class = "workimg"> <img src="./img/workimg/work3.svg" alt = "Work Image Screenshot1">
+       </div>
+    <h1>Tonic</h1>
+    <ul class = "work-ul">
+    <li><h2>CANOPY</h2></li>
+    
+    <li><h3><img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;Back End Dev &nbsp; &nbsp; <img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;2015 </h3></li>
+</ul>
+
+<p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class = "work-tags">
+    <li class="html">html</li>
+    <li class = "css">css</li>
+    <li class = "javascript">javascript</li>
+</ul>
+
+<button class="button button3">See Project</button>
+    </div>
+
+
+
+    <div class="items work4">
+        <div class = "workimg"> <img src="./img/workimg/work4.svg" alt = "Work Image Screenshot2">
+       </div>
+    <h1>Multi-Post Stories</h1>
+    <ul class = "work-ul">
+    <li><h2>CANOPY</h2></li>
+    
+    <li><h3><img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;Back End Dev &nbsp; &nbsp; <img src="./img/workimg/Counter.svg"> &nbsp; &nbsp;2015 </h3></li>
+</ul>
+
+<p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class = "work-tags">
+    <li class="html">html</li>
+    <li class = "css">css</li>
+    <li class = "javascript">javascript</li>
+</ul>
+
+<button class="button button4">See Project</button>
+    </div>
+   
+</div>
+
+<div class="about-myself">
+    <div class="about-bg">
+    <h1> 
+        About Myself</h1>
+        <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <h2> LET's CONNECT</h2>
+<div class="social-links">
+<ul>
+    <li ></li><a href="https://twitter.com/dheerajarya"><img src="./img/twitter.svg"></a>
+    </li>
+    <li ></li><a href="https://www.linkedin.com/in/dheeraj-sachdeva-502b2b8/"><img src="./img/Linkedin icon.svg"></a>
+</li>
+<li ></li><a href="#"><img src="./img/M.svg"></a>
+</li>
+<li ></li><a href="https://github.com/dheerajsachdeva"><img src="./img/github.svg"></a>
+    </li>
+    <li ></li><a href="#"><img src="./img/victory.svg"></a>
+    </li>
+    
+</ul>
+</div>
+<button class="button button5">Get my resume</button>
+
+<ul class = "skills">
+<li class = "skills-li languages" >Languages</li>
+<ul class = "javascrip-ul" >
+    <li class = "javascript-li"><div class = "java-img"><img src="./img/skills/Javasript.svg"> </div> <div class = "java-text">Javascript</div></li>
+    
+    <li class = "javascript-li"><div class = "java-img"><img src="./img/skills/HTML.svg"> </div> <div class = "java-text">HTML</div></li>
+
+    <li class = "javascript-li"><div class = "java-img"><img src="./img/skills/CSS.svg"> </div> <div class = "java-text">CSS</div></li>
+    
+   
+</ul>
```

Comment:
Your skills section looks great. however, kindly include the arrows as shown on the illustration.

Your design:
![image](https://user-images.githubusercontent.com/20554583/212427425-ad1e0c83-800c-4f8d-ba9d-4c04720f5ce8.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/212427470-3e5173d3-a526-4641-b753-98831b795dea.png)